R/suffix.R

```diff
@@ -142,8 +142,9 @@ suffix_extract <- function(domains, suffixes = NULL){
         stop("Expected column named \"suffixes\" in suffixes data.frame")
       }
     }
-    suffix_load(suffixes)
   }
+  suffix_load(suffixes)
```

Comment:
This'll automatically regenerate suffixes on every call, negating the utility of calling suffix_load in .onLoad(), no?

Maybe we should cut the suffix_load() modifications bit? I'm still noodling on how to implement it myself.